docs/guides/effects.md

```diff
@@ -2,7 +2,7 @@
 
 In statecharts, "side-effects" can be grouped into two categories:
 
-**"Fire-and-forget" effects**, which execute a synchronous side-effect with no events sent back to the statechart, or _send an event synchronously_ back to the statechart:
+**"Fire-and-forget" effects**, which execute asynchronous side-effect with no events sent back to the statechart, or _send an event synchronously_ back to the statechart:
```

Comment:
```suggestion
**"Fire-and-forget" effects**, which execute a synchronous side-effect with no events sent back to the statechart, or _send an event synchronously_ back to the statechart:
```

This would change the meaning of the sentence.